README.md

```diff
@@ -2,17 +2,30 @@
 
 This is a minimal Python package that allows users to download, query, and upload data from the IMAP Science Data Center (SDC).
```

Comment:
```suggestion
This lightweight Python package allows users to download, query, and upload data from the IMAP Science Data Center (SDC).
```

For some reason I feel that using the word 'minimal' here comes off as diminishing of the capability of this package. Users might think 'this is not going to be good enough to handle my complex query'.

Reply:
I like this suggestion, thanks!